app/assets/javascripts/components/course.cjsx

```diff
@@ -61,7 +61,7 @@ Course = React.createClass(
     user_role = if user_obj? then user_obj.role else -1
 
     if (user_role > 0 || @getCurrentUser().admin) && !@state.course.legacy && !@state.course.published
-      if CourseStore.isLoaded() && !(@state.course.submitted || @state.published)
+      if CourseStore.isLoaded() && !(@state.course.submitted || @state.published) && @state.course.type == 'ClassroomProgramCourse'
```

Comment:
`:type` is not being sent to the client. To use this, we need to add type to /app/views/courses/course.json.jbuilder